### PR TITLE
tests: testing new fedora 33 image

### PR DESCRIPTION
**Pull request by sergiocazzolato**
Also fedora 31 is not executed anymore, it remains manual and fedora 30 is removed from spread.yaml

### CODE REVIEW

**Comment by sergiocazzolato** [comment]
Main error on tests is that it is waiting to find /lib/x86_64-linux-gnu/libc.so.6 but it is located /usr/lib64/libc.so.6 producing the error:
/usr/lib/snapd/snap-exec: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /usr/lib/snapd/snap-exec)

**Comment by bboozzoo** [comment]
Pushed a commied I had stashed and totally forgotten about. This should fix the build. The changes are already part of the downstream packaging in Fedora.

**Comment by bboozzoo** [comment]
Fedora 33 is passing now.

File: spread.yaml [hunk 1]
```diff
@@ -111,13 +111,13 @@ backends:
             - debian-sid-64:
                   workers: 6
 
-            - fedora-30-64:
-                  workers: 6
-                  manual: true
             - fedora-31-64:
                   workers: 6
```
Comment (stolowski) [inline spread.yaml:115]: Aren't we dropping fedora 31 in the tests below and only testing 32 & 33 now?

Comment (stolowski) [reply spread.yaml:115]: So apparently fedora-31 is not EOL just yet (will be soon), but I'm confused about why are we removing it from systems lines in the tests then.

**Review by stolowski — APPROVED** [review]
LGTM. NB, Description/commit message needs updating to reflect dropping of fedora 31.

**Pull request merged**